.gitlab-ci-check-shell-format.yml

```diff
@@ -0,0 +1,72 @@
+# .gitlab-ci-check-shell-format.yml
+#
+# This gitlab-ci file runs the shfmt code formatter https://github.com/mvdan/sh
+# on the shell code in the repository, and fails the pipeline if the code is
+# not formatted correctly
+#
+# Add it to the project in hand through Gitlab's include functionality
+#
+# include:
+#   - project: 'Northern.tech/Mender/mendertesting'
+#     file: '.gitlab-ci-check-shell-format.yml'
+#
+# By default, it will check all files with "*.sh" extension. To change this,
+# override before_script defining your custom SHELL_SCRIPTS variable. For
+# example:
+#
+#  before_script:
+#    - SHELL_SCRIPTS=$(find mender-convert* modules configs scripts
+#            -type f
+#            -not -name "*.md"
+#            -not -path "scripts/linkbot/*"
+#            -not -wholename "scripts/test/*")
+#
+
+stages:
+  - test
+
+test:check-shell-formatting:
+  stage: test
+  needs: []
+  image: alpine:3.13
+  before_script:
+    - SHELL_SCRIPTS=$(find . -type f -name "*.sh")
+  script:
+    - apk update
+    - apk upgrade
+    - apk add --update bash git shfmt
+    - shfmt -version
+    - /bin/bash
+    - 'echo -e "Checking shell formatting for the following files:\n${SHELL_SCRIPTS}"'
```

Comment:
is this in single quotes on purpose?

Reply:
The single quotes is to make the line correctly parsed as a yaml entry (due to the `:` half way)